src/composer.ts

```diff
@@ -200,10 +200,10 @@ export interface Composer<
   t(key: Path, named: NamedValue, defaultMsg: string): string
   t(key: Path, named: NamedValue, options: TranslateOptions): string
   t(...args: unknown[]): string // for internal
-  d(value: number | Date): string
-  d(value: number | Date, key: string): string
-  d(value: number | Date, key: string, locale: Locale): string
-  d(value: number | Date, options: DateTimeOptions): string
+  d(value: number | Date | string): string
```

Comment:
When typescript 4.1 is released, we should define ISO type  use template literal types.
https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-beta/#template-literal-types